Projects/Features/SearchFeature/Sources/View/SongResultCell.swift

```diff
@@ -0,0 +1,94 @@
+import DesignSystem
+import Kingfisher
+import LogManager
+import SongsDomainInterface
+import UIKit
+import Utility
+
+final class SongResultCell: UICollectionViewCell {
+    private let thumbnailView: UIImageView = UIImageView().then {
+        $0.contentMode = .scaleAspectFill
+        $0.clipsToBounds = true
+        $0.layer.cornerRadius = 4
+    }
+
+    private let titleLabel: UILabel = UILabel().then {
+        $0.numberOfLines = 1
+        $0.font = .setFont(.t6(weight: .medium))
+        $0.textColor = DesignSystemAsset.NewGrayColor.gray900.color
+    }
```

Comment:
```suggestion
    private let titleLabel = WMLabel(
        text: "",
        textColor: DesignSystemAsset.NewGrayColor.gray900.color,
        font: .t6(weight: .medium),
        alignment: .center,
        lineHeight: UIFont.WMFontSystem.t6(weight: .medium).lineHeight,
        kernValue: -0.5
    ).then {
        $0.numberOfLines = 1
    }
```
UILabel로 되서 자간이 적용안된 곳이 상당히 많은데 WMLabel 로 변경하는 작업이 필요할것같아요

Reply:
요거는  https://github.com/wakmusic/wakmusic-iOS/pull/589 여기서 작업햇어요

Reply:
아하 굳굳